src/taxman.py

```diff
@@ -126,89 +126,150 @@ def evaluate_sell(op: transaction.Operation) -> Optional[transaction.TaxEvent]:
                         real_gain += gain
             remark = ", ".join(
-                f"{sc.sold} from {sc.op.utc_time} " f"({sc.op.__class__.__name__})"
+                f"{sc.sold} von {sc.op.utc_time} " f"({sc.op.__class__.__name__})"
                 for sc in sold_coins
             )
             return transaction.TaxEvent(
                 taxation_type,
                 taxed_gain,
                 op,
-                sell_price,
+                is_taxable,
+                sell_value,
                 real_gain,
                 remark,
             )
 
         for op in operations:
             if isinstance(op, transaction.Fee):
                 balance.remove_fee(op.change)
+                taxation_type = "Sonstige Einkünfte"
                 if self.in_tax_year(op):
                     # Fees reduce taxed gain.
-                    taxation_type = "Sonstige Einkünfte"
-                    taxed_gain = -self.price_data.get_cost(op)
-                    tx = transaction.TaxEvent(taxation_type, taxed_gain, op)
-                    self.tax_events.append(tx)
+                    is_taxable = True
+                else:
+                    taxation_type += " außerhalb des Steuerjahres"
+                    is_taxable = False
+                taxed_gain = -self.price_data.get_cost(op)
+                tx = transaction.TaxEvent(taxation_type, taxed_gain, op, is_taxable)
             elif isinstance(op, transaction.CoinLend):
-                pass
+                taxation_type = "CoinLend"
+                tx = transaction.TaxEvent(taxation_type, decimal.Decimal(), op, False)
             elif isinstance(op, transaction.CoinLendEnd):
-                pass
+                taxation_type = "CoinLendEnd"
+                tx = transaction.TaxEvent(taxation_type, decimal.Decimal(), op, False)
             elif isinstance(op, transaction.Staking):
-                pass
+                taxation_type = "Staking"
+                tx = transaction.TaxEvent(taxation_type, decimal.Decimal(), op, False)
             elif isinstance(op, transaction.StakingEnd):
-                pass
+                taxation_type = "StakingEnd"
+                tx = transaction.TaxEvent(taxation_type, decimal.Decimal(), op, False)
             elif isinstance(op, transaction.Buy):
                 balance.put(op)
+                if misc.is_fiat(op.coin):
+                    continue
+                else:
+                    taxation_type = "Kauf"
+                    cost = self.price_data.get_cost(op)
+                    price = self.price_data.get_price(
+                        op.platform,
+                        op.coin,
+                        op.utc_time,
+                        config.FIAT
+                    )
+                    remark = (
+                        f"Kosten {cost} {config.FIAT}, "
+                        f"Preis {price} {config.FIAT}/{op.coin}"
+                    )
```

Comment:
Would you rather suggest to add new columns for count, price and total values?

Reply:
Yes. Thinking of a more detailed export, it would be nice to split the "grouped" sells into single sells.

Reply:
What about modifying `evaluate_sell` so that a list of `TaxEvent` is returned instead of a joined remark list?
Should we also store asset count, price and cost as variables in `TaxEvent`? Then it's easier to create a detailed table afterwards.

Reply:
Asset, count and price are saved in the operation of the TaxEvent `tx.op`.